near-sdk-macros/src/core_impl/code_generator/ext.rs

```diff
@@ -241,7 +241,7 @@ mod tests {
             #[warn(unused)]
             pub fn method(&self) { }
         };
-        let method_info = ImplItemMethodInfo::new(&mut method, impl_type).unwrap();
+        let method_info = ImplItemMethodInfo::new(&mut method, false,impl_type).unwrap().unwrap();
```

Comment:
curious, doesn't look like cargo-fmt acted on these..

```suggestion
        let method_info = ImplItemMethodInfo::new(&mut method, false, impl_type).unwrap().unwrap();
```

Reply:
Hmm, that's interesting. Looks like a bug, wondering if I can create a minimal reproduction.